src/stateToHTML.js

```diff
@@ -80,6 +81,10 @@ const DATA_TO_ATTR = {
         attrs[dataKey] = dataValue;
       }
     }
+
+    attrs.rel = 'noopener';
+    attrs.target = '_blank';
+
```

Comment:
Why hard code it here? Change the package so you can pass these as part of the configuration.